### ISSUE TITLE
`--template` should accept URL 

### ISSUE THREAD

**Issue by ncphillips** [forestryio/create-static-site#30]
When running `create-static-site` the user should be able to pass in a URL for the template

```bash
npx create-static-site --template=<URL>
```

`static-scripts init` should grab the template project from that URL 

The existing options `hugo` and `jekyll` should be aliases to URLs.

This will make it possible for other templates to be created without changing this project. It will also reduce the size of the `static-scripts` NPM package.

**Comment by scottgallant** [comment]
Agreed, create static site should use external projects for the templates.

However, to keep setup simple, it would be ideal if create-static-site asked you to choose a template and provided you with a list of external URLS for the top templates. For example:

```
Please pick a template (use arrow keys):
Hugo (https://github.com/path-to-hugo-template)
> Jekyll (https://github.com/path-to-jekyll-template)
Gatsby (https://github.com/path-to-gatsby-template)
Other 
```

^ Choosing "Other" would prompt you to paste in a path to your template.

Curious if @chrisdmacrae has any thoughts here.

**Comment by scottgallant** [comment]
This is inspired by [Vue CLI](https://github.com/vuejs/vue-cli)

<img width="614" alt="screen shot 2018-05-02 at 6 27 22 pm" src="https://user-images.githubusercontent.com/776019/39550171-cac9d374-4e36-11e8-84c3-591657a0dd6c.png">
